### PR TITLE
Feature/get post functionality

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [x] Refactor
 
Describe the feature - fix - refactor: refactored domUpdates and scripts and tests

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test: scripts.js

Additional comments:

### CODE REVIEW

File: src/classes/Pantry.js [hunk 1]
```diff
@@ -47,55 +47,71 @@ class Pantry {
...
+  // }
+  //make this method add ingredients by positive or negative number based on cooking or shopping
+  //return the pantry array with all updates
 
-  makeShoppingList(ingredientNames) {
-    return ingredientNames.map(name => {
-    ingredientsData.forEach(ingredient => {
-        if (ingredient.name === name) {
-          name = ingredient
+  addToShoppingList(nameInput, amountInput) {
```
Comment (StevieAmb) [inline src/classes/Pantry.js:75]: Yes, it was hard work figuring this out!! Go team!

File: src/domUpdates.js [hunk 2]
```diff
@@ -0,0 +1,419 @@
+import { fetchApiData } from './apiCalls';
```
Comment (StevieAmb) [inline src/domUpdates.js:1]: Cool thing to figure out! :D

**Review by lschnell8 — APPROVED** [review]
Fantastic push and teamwork ya'll! We made great progress on this after we figured out that internet lag was causing some of the bugs we were experiencing.

File: src/classes/Pantry.js [hunk 3]
```diff
@@ -47,55 +47,71 @@ class Pantry {
...
-
-        }
-      })
-      if ((!acc.includes(pantryIngredient)) && (pantryIngredient.amount !== 0)) {
-        acc.push(pantryIngredient)
-        this.shelf = acc
-      }
-        return acc
-    }, [])
+  cookRecipe(recipe) { //-subtract
```
Comment (StevieAmb) [inline src/classes/Pantry.js:50]: Great job!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**